src/Graphics/Sprite.h

```diff
@@ -72,6 +69,8 @@ class Sprite
 
     GLint _attribPos;
     GLint _attribTex;
+	Texture* _texture;
+	Graphics::TransFlags::Trans _trans = Graphics::TransFlags::Trans::NONE;
```

Comment:
sorry,  my project settings currently are default to visual studio which means enter and copy paste will auto tab them